subsys/mpsl/Kconfig.fem

```diff
@@ -72,4 +72,12 @@ config MPSL_FEM_NRF21540_RX_GAIN_DB
 
 endif
 
+config MPSL_FEM_DEVICE_CONFIG_254
+	bool "Apply device configuration 254"
+	default y
+	help
+	  Device configuration 254 is required to conform to the requirements in
+	  section TP/154/PHY24/TRANSMIT-05 of the ZigBee IEEE 802.15.4 Test
+	  Specification.
```

Comment:
I think this message is misleading.

Config 254 __may__ be required to conform with mentioned requirements in 802.15.4 radio mode, especially when external PA is included in the device.
Moreover, config 254 __may__ be required to improve RX blocking performance, especially when external LNA is included in the device.

Reply:
@hubertmis, could you, please, check if description is now correct?